bot/logger/logger.py

```diff
@@ -0,0 +1,60 @@
+"""Logger Module. Configures the logger for the discord library and the logger for custom code.
+It also provides the decorator for logging command calls.
+"""
+import functools
+import logging
+
+import discord
+
+from bot import constants
+
+# configure logger for bot
+log = logging.getLogger("bot")
+log.setLevel(logging.INFO)
+file_handler = logging.FileHandler(filename=constants.LOG_FILE_PATH, encoding='utf-8', mode='w')
```

Comment:
Maybe add some blank lines and group the existing lines of code to make it more readable?

Reply:
I had empty lines between the different handler registrations for bot code and discord.py lib code